.github/actions/test-dotnet/action.yml

```diff
@@ -47,18 +47,24 @@ inputs:
       Determines how many UI tests will run in parallel. Affects both xUnit's maxParallelThreads configuration and the
       MaxParallelTests configuration of Lombiq UI Testing Toolbox. If set to "-1" then the configuration coming from the
       code won't be overridden.
-  ui-test-artifact-retention-days:
+  artifact-retention-days:
```

Comment:
Please mention this breaking change in the `Readme.md` and/or the PR description to make it visible.

Also, why do we remove the `ui.test-` prefix? We are using prefixes in similar purpose parameters names. See:
- `build-dotnet` -> `binary-log-artifact-retention-days`
- `publish-nuget` -> `nuget-artifact-retention-days`

Reply:
Added this to the PR title; I expect the impact to be low to consumers (none of our projects set it either).

I chose the name `artifact-retention-days` here in the action because it's about all artifacts of the action. However, in the reusable workflows, where more kinds of artifacts can be generated, it's `test-artifact-retention-days` (and the description references testing too), so there is a distinction with any other kind of artifact.